packages/meta/collection.yaml

```diff
@@ -83,7 +83,7 @@ packages:
     requires:
       - category: toolchain
         name: cosign
-        version: "<1.4"
+        version: ">=1.4"
```

Comment:
```suggestion
        version: ">1.3.1"
```

Reply:
Maybe we should also add a comment on why? (and link to the issues.. so we don't forget in the future?)

Reply:
Trying to see first why locally validation passes but not in ci...